src/main/resources/wa-task-configuration-wa-wacasetype.dmn

```diff
@@ -139,7 +139,7 @@ else if caseData.appealType = "revocationOfProtection" then "Revocation" else ""
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_1a8bdg5">
-          <text>"processApplication","reviewAppealSkeletonArgument","followUpOverdueRespondentEvidence","validateMandatoryTaskAttributesDuringInitiation","validateMandatoryTaskAttributesDuringReconfiguration","functionalTestTask1","calculateDueDate","endToEndTask","reconfigTaskAttributesTask","followUpOverdue","functionalTestTask2","reviewTheAppeal"</text>
+          <text>"processApplication","reviewAppealSkeletonArgument","followUpOverdueRespondentEvidence","validateMandatoryTaskAttributesDuringInitiation","validateMandatoryTaskAttributesDuringReconfiguration","functionalTestTask1","calculateDueDate","endToEndTask","reconfigTaskAttributesTask","followUpOverdue","functionalTestTask2","reviewTheAppeal","secondTask"</text>
         </inputEntry>
```

Comment:
Can't see the full context here but can be try and use more descriptive names?  Maybe hinting at the test case?